examples/java/src/test/java/org/apache/beam/examples/TfIdfIT.java

```diff
@@ -16,34 +16,41 @@
  * limitations under the License.
  */
 
-package org.apache.beam.runners.spark;
+package org.apache.beam.examples;
 
 import org.apache.beam.examples.complete.TfIdf;
+import org.apache.beam.runners.spark.EvaluationResult;
+import org.apache.beam.runners.spark.SparkPipelineOptions;
+import org.apache.beam.runners.spark.SparkRunner;
 import org.apache.beam.sdk.Pipeline;
 import org.apache.beam.sdk.coders.StringDelegateCoder;
 import org.apache.beam.sdk.options.PipelineOptionsFactory;
 import org.apache.beam.sdk.testing.PAssert;
+import org.apache.beam.sdk.testing.TestPipeline;
 import org.apache.beam.sdk.transforms.Create;
 import org.apache.beam.sdk.transforms.Keys;
 import org.apache.beam.sdk.transforms.RemoveDuplicates;
 import org.apache.beam.sdk.values.KV;
 import org.apache.beam.sdk.values.PCollection;
 
 import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
 
 import java.net.URI;
 import java.util.Arrays;
 
 /**
- * A test based on {@code TfIdf} from the SDK.
+ * End-to-end tests of {@link TfIdf}.
  */
-public class TfIdfTest {
+@RunWith(JUnit4.class)
+public class TfIdfIT {
 
   @Test
-  public void testTfIdf() throws Exception {
-    SparkPipelineOptions opts = PipelineOptionsFactory.as(SparkPipelineOptions.class);
-    opts.setRunner(SparkRunner.class);
-    Pipeline pipeline = Pipeline.create(opts);
+  public void testE2ETfIdfSpark() throws Exception {
+    SparkPipelineOptions options = PipelineOptionsFactory.as(SparkPipelineOptions.class);
```

Comment:
Could you use `PipelineOptionsFactory.fromArgs` to be runner agnostic ? Is there a benefit in that ? maybe apply the same to all runners ?